### PR TITLE
[v6r15] Fix for API and various fixes for tests

### PR DESCRIPTION
**Pull request by fstagni**

### CODE REVIEW

File: tests/Integration/DataManagementSystem/TestClientFTS3.py [hunk 1]
```diff
@@ -7,245 +7,246 @@
 from DIRAC.Core.Base.Script import parseCommandLine
 parseCommandLine()
 
-import unittest
```
Comment (chaen) [inline tests/Integration/DataManagementSystem/TestClientFTS3.py]: hum... ?? :-) You don't want to test FTS3 ? :-)

File: tests/System/FTSGraphTests.py [hunk 2]
```diff
@@ -1,82 +0,0 @@
-########################################################################
```
Comment (chaen) [inline tests/System/FTSGraphTests.py]: why as well ?

File: tests/Integration/DataManagementSystem/TestClientFTS3.py (hunk 1 above)
Comment (fstagni) [inline tests/Integration/DataManagementSystem/TestClientFTS3.py]: I do, but there's nothing called FTS3Client (anymore) so I don't what to do. I guess this your stuff now :)

File: tests/System/FTSGraphTests.py (hunk 2 above)
Comment (fstagni) [inline tests/System/FTSGraphTests.py]: IIUC this tests some abandoware

File: tests/Integration/DataManagementSystem/TestClientFTS3.py (hunk 1 above)
Comment (chaen) [inline tests/Integration/DataManagementSystem/TestClientFTS3.py]: it has not yet been merged

File: tests/System/FTSGraphTests.py (hunk 2 above)
Comment (chaen) [inline tests/System/FTSGraphTests.py]: it tests mechanics for FTS2, so for us it doesn't matter no, I don't know about others.

**Comment by atsareg** [comment]
Review OK

**Comment by chaen** [comment]
I don't think it is ok, the tests of FTS3 should not be commented.

**Comment by atsareg** [comment]
Can you please decide what to do with the FTS3 tests together with Federico ? and make another PR then, I have merged this one already.

**Comment by chaen** [comment]
I re-added the test in my PR for FTS3 in v6r16

**Pull request merged**